prosemirror-jdita/src/plugin.ts

```diff
@@ -5,28 +5,69 @@ import { toggleMark, newLine, hasMark, insertNode, insertImage, InputContainer }
 import { Command } from "prosemirror-commands";
 import { redo, undo } from "prosemirror-history";
 
+/**
+ * Retrieve the DOM element with ID "editor" and assign it to a constant targetNode
+ * This is the entire DOM node of the Prosemirror editor that will be observed for DOM mutations
+ */
 const targetNode = document.getElementById('editor');
+
+/**
+ * Set the Observer to watch for changes being made to the editor DOM tree
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver}
+ * for more info on the MutationObserver interface
+ */
+// make sure the editor element exists
 if (targetNode) {
+  // Options for the observer (which mutations to observe)
   const config = { attributes: false, childList: true, subtree: true };
+  // Callback function to execute when mutations are observed
   const callback: MutationCallback = function(mutationsList) {
+    // loop through the mutations
```

Comment:
Redundant comment. The code's intention is clear.

Reply:
Fixed with f092276.